libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Visualizations/FunnelChart/FunnelChartScenario.tsx

```diff
@@ -0,0 +1,21 @@
+// (C) 2007-2023 GoodData Corporation
+import React from "react";
+import { FunnelChart } from "@gooddata/sdk-ui-charts";
+import * as BearMDObjects from "../../../../../../reference_workspace/workspace_objects/goodsales/current_reference_workspace_objects_bear";
+import * as TigerMDObjects from "../../../../../../reference_workspace/workspace_objects/goodsales/current_reference_workspace_objects_tiger";
+
+type MDObjectsType = typeof TigerMDObjects & typeof BearMDObjects;
+
+export const MDObjects = (
+    process.env.SDK_BACKEND === "TIGER" ? TigerMDObjects : BearMDObjects
+) as MDObjectsType;
```

Comment:
As the previous comment, we don't need to run UI SDK test for both of backend, so we don't need to IF, ELSE here to select the MDObject